File: Activities/Activity 6/Activity6.cpp
/**
 * Activity 6: Selection, and the failbit
 * Grant Loofbourrow
 * Nathan Saugstad
 * Ana Solis
 */
#include <iostream>

int main() {

    int number, status;
    std::cout << "Enter integer to test: ";
    std::cin >> number; //take integer for switch

    if(std::cin.fail()) {

        status = 1;

        std::cin.clear();
        char c;
        std::cin.get(c);
        std::cout << "Oops, you entered: " << c << std::endl;

    } else {

        status = 0;

        switch(number) {
            case 1:
                std::cout << "good" << std::endl;
                break;
            case 2:
                std::cout << "better" << std::endl;
                break;
            case 3:
                std::cout << "best" << std::endl;
               break;
            default:
                std::cout << "value out of range" << std::endl;
                break;
        }

    }

    return status;

}

File: Activities/Activity 6/Activity6.java
import java.util.Scanner;

public class Activity6 {

	public static void main(String[] args) {
		
		int number = 0;
		Scanner s = new Scanner(System.in);
		
		System.out.println("Enter integer to test: ");
		String input = s.next();
		s.close();
		
		try {
			number = Integer.parseInt(input);
		} catch(Exception e) {
			System.out.println("Oops you entered: " + input);
			System.exit(1);
		}
		
		switch(number) {
			case 1:
				System.out.println("good");
				break;
			case 2:
				System.out.println("better");
				break;
			case 3:
				System.out.println("best");
				break;
			default:
				System.out.println("value out of range");
				break;
		}
		
	}
	
}